deployments/ansible/molecule/config/windows.yml

```diff
@@ -31,13 +31,13 @@ platforms:
     instance_raw_config_args: *vagrant_args
   - name: "2019"
     box: gusztavvargadr/windows-server-2019-standard
-    box_version: 1809.0.2305
+    box_version: 1809.0.2312
```

Comment:
maybe this needs to be in a different PR

Reply:
I don't think this is an option: it seems that the version 2305 was removed https://app.vagrantup.com/gusztavvargadr/boxes/windows-server-2019-standard-core/versions/1809.0.2305 - it seems that the oldest available is 2307.

Reply:
ouch. But that can go into a separate PR right now to get main back to green?